package.json

```diff
@@ -44,52 +44,54 @@
     "useTabs": true,
     "tabWidth": 2
   },
-  "lint-staged": {
-    "{src,test}/**/*.{css,js,json}": [
+  "simple-git-hooks": {
+    "pre-commit": "npx nano-staged"
+  },
+  "nano-staged": {
+    "**/*.{css,js,jsx,json}": [
```

Comment:
The lint-staged config was pretty restricted, so we're probably going to see a lot more formatting corrections pop up in the future here, though I think that's alright.